### DISCUSSION TITLE
Add Bookmark or Preset buttons ?

### DISCUSSION THREAD

**Discussion by ceejayemm** [rern/rAudio#1078] [Feature Requests]
From the Library we can currently add a bookmark to an album and it appears on the Library main screen.  Would it be possible to also allow Web Radio stations to be bookmarked in a similar way ?  This would go some way to helping out my wife who finds the current means of charging web radio stations difficult - it would at least cut down the number of steps she needs to do. Any such Bookmark should automatically function in a 'Replace / Play' mode so that as soon as the Bookmark is selected the items starts to play.

A further development of this would be to add a set of 'Preset' buttons to the main Playback screen, this would very definitely need to be optional to maintain current functionality if required.  If the 'Presets' option is selected then allow a small number of 'buttons' (say, up to a maximum of 5) to be configured, each button could then be configured to point to any item ALREADY existing in the Library eg albums, web radio stations, bookmarks (?)  or playlists. Only those 'Preset' buttons which have been configured should then show on the main Playback screen thus helping to keep the Playback screen tidy.

For those who need it this would go towards improving the accessibility of rAudio-1.  Just some thoughts on future developments for this great product.

Chris

**Comment by R101G** [comment]
@ceejayemm 

I too would like to have a bookmark option for radio streams, but I am not so sure that changing the playback page would be a good idea.
